pyproject.toml

```diff
@@ -196,7 +196,7 @@ duckdb = [
   "sqlalchemy",
   "sqlalchemy-views",
 ]
-flink = []
+flink = ["pyspark"]
```

Comment:
Hm, why does the flink backend depend on pyspark?

Reply:
I just added it as a placeholder initially, since I couldn't actually add `pyflink`. 🙈 Very initially, thought would start with executing the generated SQL in Spark until could use PyFlink, but have a workaround for now, so probably should revert this.